extra/pattern/pyramid.js

```diff
@@ -0,0 +1,56 @@
+
+export default function pyramid(row,type){
+    if(type=='halfpyramid'){
+        let n = row;
+        let string = "";
+        for (let i = 1; i <= n; i++) {
```

Comment:
@arjavjainsuccessive, please mention the valid name instead of i, j, and string.